Cope with some problem, possibly related to the old version of curses in OpenBSD

Index: auto.def
--- auto.def.orig
+++ auto.def
@@ -754,9 +754,9 @@ cc-with [list -libs -L$ncurses_prefix/$::libdir_tail] 
     user-error "Unable to find ncursesw library"
   }
 
-  foreach f {bkgrndset setcchar} {
-    cc-check-function-in-lib $f $ncurses_lib
-  }
+#  foreach f {bkgrndset setcchar} {
+#    cc-check-function-in-lib $f $ncurses_lib
+#  }
 }
 
 # Locate the directory containing ncurses.h
